index.html

```diff
@@ -0,0 +1,205 @@
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <a class="moreabout" href="#">See Project</a>
+        </div>
+
+        <!-- 3rd work! -->
+        <div class="works">
+          <img src="images/works/work3.svg" alt="" />
+          <h2>Tonic</h2>
+          <ul>
+            <li class="worksubtitle">CANOPY</li>
+            <li class="bullets">Back End Dev</li>
+            <li class="bullets">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <a class="moreabout" href="#">See Project</a>
+        </div>
+
+        <!-- 4rt work! -->
+        <div class="works">
+          <img src="images/works/work4.svg" alt="" />
+          <h2>Tonic</h2>
+          <ul>
+            <li class="worksubtitle">CANOPY</li>
+            <li class="bullets">Back End Dev</li>
+            <li class="bullets">2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li class="lang">html</li>
+            <li class="lang">css</li>
+            <li class="lang">javaScript</li>
+          </ul>
+          <a class="moreabout" href="#">See Project</a>
+        </div>
+      </section>
+
+      <!-- About me section! -->
+      <section class="aboutme">
+        <div class="title">About Myself</div>
+        <div class="txt">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </div>
+        <div class="socialresume">
+          <h3 class="hsocial">LET'S CONNECT</h3>
+          <ul>
+            <li class="social">
+              <a href="#"><img src="images/twitter.svg" alt="" /></a>
+            </li>
+            <li class="social">
+              <a href="#"><img src="images/Linkedin.svg" alt="" /></a>
+            </li>
+            <li class="social">
+              <a href="#"><img src="images/M.svg" alt="" /></a>
+            </li>
+            <li class="social">
+              <a href="#"><img src="images/cat.svg" alt="" /></a>
+            </li>
+            <li class="social">
+              <a href="#"><img src="images/Y.svg" alt="" /></a>
+            </li>
+          </ul>
+          <div class="moreabout"><a href="#">Get my resume</a></div>
+        </div>
+        <div class="resume">
+          <div>
+            <h2 class="subt">Languages</h2>
+            <img src="images/works/Uniona.png" alt="" />
+          </div>
```

Comment:
- [ ] Please make sure to wrap all "Languages", "Frameworks" and "Skills" into a `ul` tag, with the correspondent `li` tag each. As a result of this change, you will have 2 nested `ul`'s, one wrapping "Frameworks, Languages and Skills" and the other one wrapping "HTML", "CSS" and "JavaScript".

Reply:
done!